scripts/conf/treode.init

```diff
@@ -0,0 +1,161 @@
+#!/bin/sh
+# Copyright 2014 Treode, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+### BEGIN INIT INFO
+# Provides:          treode
+# Required-Start:    $network $local_fs
+# Required-Stop:
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Treode server
+# Description:       Treode server
+### END INIT INFO
+
+# Author: treode@treode.com
+
+# PATH should only include /usr/* if it runs after the mountnfs.sh script
+PATH=/sbin:/usr/sbin:/bin:/usr/bin
+
+#JAVA_HOME=$(readlink -f /usr/bin/java | sed "s:/bin/java::")
+JAVA_HOME=/usr/lib/jvm/default-java
+DESC=treode            
+NAME=treode            
+DAEMON="$JAVA_HOME/bin/java"
+TREODEKV_DB=/opt/treode/db/store.3kv
+DAEMON_ARGS="-jar /opt/treode/lib/finagle-server-0.2.0.jar serve -solo  -log.output=/opt/treode/db/treode.log $TREODEKV_DB"
```

Comment:
The version number 0.2.0 appears throughout frequently. We need an easy way to upgrade versions.